From 4e3de9c4d724b8449323c920de2c8c02bce1ccd5 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Wed, 15 Jan 2025 20:53:37 +0100
Subject: [PATCH] Python 3.9 fixes

---
 olmo/util.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/olmo/util.py b/olmo/util.py
index 1cf758b..03ed2d7 100644
--- a/olmo/util.py
+++ b/olmo/util.py
@@ -715,7 +715,7 @@ def _http_get_bytes_range(scheme: str, host_name: str, path: str, bytes_start: i
 
 
 def save_hf_dataset_to_disk(
-    dataset: datasets.DatasetDict | datasets.Dataset,
+    dataset: Union[datasets.DatasetDict, datasets.Dataset],
     hf_path: str,
     name: Optional[str],
     split: str,
